cluster/images/hyperkube/static-pods/master-multi.json

```diff
@@ -12,13 +12,15 @@
       "name": "controller-manager",
       "image": "REGISTRY/hyperkube-ARCH:VERSION",
       "command": [
-              "/hyperkube",
-              "controller-manager",
-              "--master=127.0.0.1:8080",
-              "--service-account-private-key-file=/srv/kubernetes/server.key",
-              "--root-ca-file=/srv/kubernetes/ca.crt",
-              "--min-resync-period=3m",
-              "--v=2"
+        "/hyperkube",
+        "controller-manager",
+        "--master=127.0.0.1:8080",
```

Comment:
If I understand you correctly, you suggest to make /srv/kubernetes available on the host. So, e.g. a user could change the default password of admin. Sound good to me.